bedrock/products/templates/products/landing.html

```diff
@@ -126,7 +126,7 @@ <h2 class="mzp-c-picto-heading"><a href="https://relay.firefox.com/{{ referrals
     {% if LANG.startswith('en-') %}
       {% set monitor_link = url('products.monitor.landing') %}
     {% else %}
-      {% set monitor_link = "https://monitor.mozilla.org/?" + utm_params %}
+      {% set monitor_link = "https://monitor.mozilla.org/" %}
```

Comment:
@alexgibson on L37 there's a `referrals` var that looks like what we would use as UTMs. Maybe we can use that here (or rename the var declaration?)

{% set referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=products' %}

Reply:
Ah good spot, updated!